chromebook-setup.sh

```diff
@@ -408,7 +408,7 @@ create_fit_image()
             fi
          fi
 
-	 local initrd_option=""
+        local initrd_option="-i arch/${ARCH}/boot/initramfs-$kernel_version.img"
          if [ -n "$INITRD" ]; then
```

Comment:
If this get's merged, would there be need for this check again?

Reply:
yes, because the $INITRD check is if to determine if a user passed an initrd to be used as an `--initrd=INITRD` option. I believe this should get more precedence over a generated initrd. In other words, we should only generate it if the user didn't specify an initrd to be used.